src/css/controls/imports/slider.less

```diff
@@ -161,8 +161,17 @@
     }
 
     .jw-flag-touch & {
-        .rect(100%, 44px);
         align-items: flex-end;
         padding-bottom: 5px;
```

Comment:
These styles are stragglers. They don't seem to do anything now. Their purpose was to push the slider back down and that isn't necessary if the `::before` is absolute.

```css
align-items: flex-end;
padding-bottom: 5px;
```